### ISSUE TITLE
Integration tests failed for tests with JHI_ENTITY=sqlfull

### ISSUE THREAD

**Issue by murdos** [jhipster/generator-jhipster#10762]
##### **Overview of the issue**

Current integration tests failed for tests with JHI_ENTITY=sqlfull

```
##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

Can't add invalid relationship. Error: In the Many-to-Many relationship from TestCustomTableName to User, only bidirectionality is supported for a Many-to-Many relationship.

ERROR! Error: 

Error while parsing entities to JDL

The command "$JHI_SCRIPTS/14-jhipster-info.sh" failed and exited with 1 during .
```

##### **Motivation for or Use Case**

Unit and integration tests on master should not fail

##### **Reproduce the error**

See https://travis-ci.org/jhipster/generator-jhipster/jobs/610557997

##### **Browsers and Operating System**

-   [X] Checking this box is mandatory (this is just to show you read everything)

**Comment by mshima** [comment]
Seems that jhipster 6.5.0 added a verification that a many-to-many relationship must be bidirectional.

User is not customizable so, this should be removed:
https://github.com/jhipster/generator-jhipster/blob/a3e51de84d320e822d97700b577ff7873164718d/test-integration/samples/.jhipster/TestCustomTableName.json#L37-L43

**Comment by pascalgrimaud** [comment]
thanks @mshima for the analyze
I'm doing the PR to remove this relationship in our entities

**Comment by pascalgrimaud** [comment]
fixed by jhipster-core v6.0.4
